tests/services/analysis_starter/test_analysis_starter.py

```diff
@@ -39,3 +51,92 @@ def test_microsalt_analysis_starter_start_available_error_handling(
 
     # THEN it should have exited with the expected value
     assert succeeded == expected_exit
+
+
+def test_rnafusion_start(
+    cg_context,
+    http_workflow_launch_response,
+    mocker: MockerFixture,
+):
+    analysis_starter: AnalysisStarter = AnalysisStarterFactory(
+        cg_context
+    ).get_analysis_starter_for_workflow(Workflow.RNAFUSION)
+    # GIVEN a store containing a case
+    mock_store = create_autospec(Store)
+    analysis_starter.store = mock_store
+    analysis_starter.input_fetcher.status_db = mock_store
+    configurator: NextflowConfigurator = analysis_starter.configurator
+    configurator.store = mock_store
+    configurator.sample_sheet_creator.store = mock_store
+    configurator.config_file_creator.store = mock_store
+    analysis_starter.tracker.store = mock_store
```

Comment:
I see that the same store has to be added to several parts of the flow. Isn't it easier to add the mock store in a AnalysisStarter fixture coming out from a factory?